.editorconfig

```diff
@@ -7,6 +7,6 @@ end_of_line = lf
 insert_final_newline = true
 trim_trailing_whitespace = true
 
-[{harness/*.js,output/*.js,README.md,package.json,.travis.yml}]
+[{*/*.js,README.md,package.json,.travis.yml}]
```

Comment:
I'm not the one to know the syntax here, but maybe it should be `**/*.js` instead of `*/*.js`?